src/plugins/embeddable/public/lib/panel/panel_header/panel_options_menu.tsx

```diff
@@ -47,15 +55,29 @@ export class PanelOptionsMenu extends React.Component<PanelOptionsMenuProps, Sta
     this.state = {
       actionContextMenuPanel: undefined,
       isPopoverOpen: false,
+      showNotification: false,
     };
   }
 
   public async componentDidMount() {
     this.mounted = true;
     this.setState({ actionContextMenuPanel: undefined });
     const actionContextMenuPanel = await this.props.getActionContextMenuPanel();
+    const showNotification = actionContextMenuPanel.actions.some(
+      (action) => action.showNotification
```

Comment:
Nice generic addition to have the action register a `showNotification` boolean! :fire: